app/javascript/components/molecules/Loading.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
I would rename this file from Loading.**jsx** to Loading.**js** since that is what we are using for all the jsx files.